CHANGELOG.md

```diff
@@ -2,6 +2,13 @@ Change Log
 ==========
 
 
+x.x.x
```

Comment:
Correct. I think it's easier to progressively fill in the changes as we go rather than trying to go back and find them all when we're finalizing a release.

Reply:
Yeah, updating the CHANGELOG.md with each PR is preferred. Just wondered about the "x.x.x"  rather than using a next potential release version. But this is fine, since we don't do much with this project and haven't planned a next release to be major/minor/bug.